plugins/base/src/test/kotlin/content/annotations/SinceKotlinTest.kt

```diff
@@ -18,8 +28,208 @@ class SinceKotlinTest : BaseAbstractTest() {
         }
     }
 
+    @BeforeEach
+    fun setSystemProperty() {
+        System.setProperty("dokka.SinceKotlin", "true")
+    }
+    @AfterEach
+    fun clearSystemProperty() {
+        System.setProperty("dokka.SinceKotlin", "")
+    }
```

Comment:
Does it have to be set and unset for each test? Can it be substituted with `@BeforeAll` and `@AfterAll`?

Reply:
Some tests can unset the property.